include/devicetree/fixed-partitions.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file
+ * @brief Flash Devicetree macro public API header file.
+ */
+
+/*
+ * Copyright (c) 2020, Linaro Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DEVICETREE_FIXED_PARTITION_H_
+#define ZEPHYR_INCLUDE_DEVICETREE_FIXED_PARTITION_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup devicetree-fixed-partition Devicetree Fixed Partition API
+ * @ingroup devicetree
+ * @{
+ */
+
+/**
+ * @brief Get fixed partition that matches label
```

Comment:
I would like the documentation to be more clear that this is the `label` property in the node, not a node label attached to the node.

Perhaps: 
* `@brief Get fixed partition with "label" property that matches label`
* `@param label lowercase-and-underscores value for label property`

Applies to both macros in this patch.

Reply:
Update the comments.